iot-hub/Quickstarts/InvokeDeviceMethod/Program.cs

```diff
@@ -19,15 +20,40 @@ internal class Program
         private static ServiceClient s_serviceClient;
         
         // Connection string for your IoT Hub
-        // az iot hub show-connection-string --hub-name {your iot hub name} --policy-name service
+        // az iot hub connection-string show --policy-name service --hub-name {YourIoTHubName}
+        // Override by passing service connection string as first parameter in command line
         private static string s_connectionString = "{Your service connection string here}";
```

Comment:
You can get rid of this class member too. It is not a good practice to make customers edit a code sample just to run it, so we stick with program parameters, and if desirable an environment variable option.